### ISSUE TITLE
Zukitwo XFCE panel does not blink on “urgent” windows

### ISSUE THREAD

**Issue by casta** [lassekongo83/zuki-themes#77]
Usually, windows with the urgency wm_hint set are blinking in the panel (so that user can see it’s requirring attention). But with Zukitwo, it’s not.

This is the workaround I use to make the window blink. I think it worth being upstream:
```
$ cat ~/.themes/Zukitwo/gtk-2.0/gtkrc
include "/usr/share/themes/Zukitwo/gtk-2.0/gtkrc"
# this is a fix for xfce panel not showing blinking windows requiring attention
pixmap_path "/usr/share/themes/Zukitwo/gtk-2.0/"
style "panel-fix" = "theme-panel-buttons" {
    engine "pixmap" {
        image {
            function    = BOX
            recolorable = TRUE
            state       = NORMAL
            file        = "assets/panel-btn-hover.png"
            border      = { 2, 2, 0, 0 }
            stretch     = TRUE
        }
    }
}
widget "*Xfce*Panel*Button*" style:highest "panel-fix"
```

**Comment by lassekongo83** [comment]
Your fix breaks the prelight state and the normal state of the buttons.

**Comment by casta** [comment]
I do not see any visual regression with this. But if you have something else to suggest, I would happy to test.
I’m really not used to gtk2 theme writing, this is the result of some empirical tests.

**Comment by lassekongo83** [comment]
Do you have any way of testing the blinking in the panel? A simple command I can call or something.

**Comment by casta** [comment]
Yes, you can do that with wmctrl: `wmctrl -r "Mozilla Firefox" -b add,demands_attention`
The pattern in `-r` option must match the title of the window you want to set the WM_URGENT hint.

**Comment by lassekongo83** [comment]
There's no easy fix for this that doesn't break the look. Panel buttons will always look like they have the prelight state. The only way to fix it would be to completely redesign the panel to not use the pixmap engine. Simply by copying Greybird's panel or something.

**Comment by Hund** [comment]
It would be nice if I could be solved somehow. I never notice when people write to me on Gajim. :(

**Comment by Hund** [comment]
Great job as always! :D

**Comment by Hund** [comment]
I'm using xfce4-panel 4.18.2, and I have this issue.
